mantidproject/mantid
# Display relative peak intensities in InstrumentView

This addresses #13907.

**Testing instructions**
1. Use the following script to create a workspace and a list of predicted peaks:

```
lattice_parameters = {'a': 5.431, 'b': 5.431, 'c': 5.431 }

CreateSimulationWorkspace(Instrument='WISH', BinParams='0,10000,20000',
                                        OutputWorkspace='test', UnitX='TOF')

SetUB(Workspace='test', u='12,1,1', v='0,4,9', **lattice_parameters)

cs = CrystalStructure('{0} {1} {2}'.format(*lattice_parameters.values()), 'F d -3 m', 'Si 0 0 0 1.0 0.01')
f = mtd['test']
f.sample().setCrystalStructure(cs)

PredictPeaks(InputWorkspace='test', CalculateStructureFactors=True,
                    MinDSpacing=0.5, MaxDSpacing=100, OutputWorkspace='predicted',
                    ReflectionCondition='All-face centred')
```
1. Open the instrument view on the `test` workspace.
2. Drag the peaks workspace onto the instrument display. All spots should have the same size.
3. Toggle the "Indicate relative intensity" option in the "Peak options" menu of the "Render" tab and notice that the spot sizes are now different. Make sure that turning it on and off works properly. When you change the `CalculateStructureFactor` option to `False`, all intensities will be zero and there should be no difference between the two options.
4. Make sure that the state of this option is preserved across Mantid restarts.
